pytripgui/app_logic/viewcanvas.py

```diff
@@ -10,8 +10,8 @@ def __init__(self):
     def widget(self):
         return self.viewcanvas_view.widget()
 
-    def set_patient(self, patient):
-        self.plot_cont.set_patient(patient)
+    def set_patient(self, patient, state=None):
+        return self.plot_cont.set_patient(patient, state)
```

Comment:
This return statement is ambiguous. Please remove it, and add method named get gui_state instead

Reply:
def get_gui_state method(self):
    return self.plot_cont.get_projection_selector()